packages/blocks/src/block-manager/index.tsx

```diff
@@ -1,23 +1,35 @@
-import {
-	type BlockData,
-	type BlockTreeNode,
-	type BlockDefinitions,
-} from '@snapwp/core';
 import defaultBlockDefinitions from '@/blocks';
 import flatListToHierarchical from '@/utils/flat-list-to-hierarchical';
 
+export interface BlockData {
+	type: keyof typeof defaultBlockDefinitions;
```

Comment:
The user is more likely to be creating components for blocks we don't bundle than for blocks we do.

Reply:
Updated the type of `type` to `string`